modules/virt-adding-vtpm-to-vm.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-vtpm-devices.adoc
+:_content-type: PROCEDURE
+[id="virt-adding-vtpm-to-vm_{context}"]
+= Adding a vTPM device to a virtual machine
+
+Adding a vTPM device to a virtual machine (VM) stores secrets for that VM.
+
+[NOTE]
+====
+vTPM implementation is non-persistent. vTPM discards stored secrets
+after each shutdown.
+====
+
+.Procedure
+
+. Edit the YAML manifest to add a vTPM device.
++
+[source,YAML]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-windows
+  name: vmi-windows
+spec:
+  domain:
+    devices:
+      tpm: {} <1>
+      disks:
+      - disk:
+          bus: sata
+        name: pvcdisk
+      interfaces:
+      - masquerade: {}
+        model: e1000
+        name: default
+...
+----
+<1> The vTPM device
```

Comment:
Please make this callout a bit more descriptive (and make sure it ends with punctuation).

Reply:
Please also add a second step where the user applies the YAML to the cluster.